include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -174,20 +174,34 @@ make_iter_mode(const _Iterator& __it) -> decltype(iter_mode<outMode>()(__it))
 }
 
 // set of templated classes to name kernels
+// Shoten the names used in kernel generator to potentially improve compilation time
+// Add aliases for such kernel names for readability
 template <typename... _Name>
-class __parallel_reduce_kernel;
+class __prk;
 
 template <typename... _Name>
-class __parallel_scan_local_kernel;
+using __parallel_reduce_kernel = __prk<_Name...>;
 
 template <typename... _Name>
-class __parallel_scan_group_kernel;
+class __pslk;
+
+template <typename... _Name>
+using __parallel_scan_local_kernel = __pslk<_Name...>;
+
+template <typename... _Name>
+class __psgrk;
+
+template <typename... _Name>
+using __parallel_scan_group_kernel = __psgrk<_Name...>;
```

Comment:
I'm not sure about identificators as an abbreviations:
1) the abbreviations are applicable, but if it a kind of more or less standard term, like USM or, CPU
2) it  is not readable at all - "psgrk"...

Reply:
The reason is to make kernel name shorter to reduce compilation time. What do you suggest?

For readability the aliases remain with the old names.

Reply:
Discussed offline. Make groups division clearer in the code. More information was added to the comments.